bukuserver/README.md

```diff
@@ -20,33 +20,28 @@ So be sure to have `python3`, `python3-pip` , `python3-dev`, `libffi-dev` packag
 #### Dependencies
 
 ```
-$ python3 -m pip install --user --upgrade pip
-$ python3 -m pip install --user virtualenv
-$ python3 -m virtualenv env
```

Comment:
Installing `virtualenv` is no longer necessary: the `venv` module is included in basic Python installation (and accomplishes the same result)